spec/requests/api/v1/merchants/item_intelligence_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Item intelligence endpoints' do
+  describe 'GET /api/v1/merchants/most_items?quantity=x' do
+    before do
+      merchants = create_list(:merchant, 5)
+      (merchants.length - 1).times do |i|
+        invoices = create_list(:invoice, i + 1, merchant: merchants[i])
+        invoices.each do |invoice|
+          create(:transaction, invoice: invoice)
+          create_list(:invoice_item, i + 1, invoice: invoice)
+        end
+      end
+    end
+
+    it 'returns top 3 merchants with most items sold' do
+      get '/api/v1/merchants/most_items?quantity=5'
```

Comment:
@Dpalazzari Yeah, this was a terrible test! Cost of early morning work. Thanks for catching this, the tests were passing because I wasn't even testing the API response. This is now fixed.